library/ceph_volume_zap.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/python
+import datetime
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.0',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: ceph_volume_zap
+
+short_description: Zap devices used by ceph OSDs with ceph-volume
+
+description:
+    - Using the ceph-volume utility available in Ceph this module
+      can be used to zap devices used by ceph OSDs.
+    - These devices, partitions or logicial volumes can then be reused.
+    - All partitions and logical volumes will be kept intact.
```

Comment:
This is not true all the time. Logical volumes may be destroyed if `destroy: True` is used